src/spaces/hsl.js

```diff
@@ -79,6 +79,7 @@ Color.defineSpace({
 	},
 
 	instance: {
+		// eslint-disable-next-line no-unused-vars
```

Comment:
In general I'm not a huge fan of these linting directives scattered throughout the codebase. It couples code and tooling too much, and if they are needed, it may indicate that the rule is too restrictive.
In this case, it looks like [`ignoreRestSiblings`](https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings) is what we need.